.gitignore

```diff
@@ -103,8 +103,9 @@ ENV/
 # PyCharm
 .idea/
 
-# ML Model
-model/
+# Rekcurd
+rekcurd-model/
```

Comment:
this example run as local mode, so LOCAL_MODEL_DIR is `model/`, not `rekcurd-model` .
(so you should ignore  `model/` not `rekcurd-model/`

Reply:
I have added `model/`. `rekcurd-model/` is also used when we don't use `local` mode.